201-front-door-standard-premium-storage-static-website/modules/storage-static-website.bicep

```diff
@@ -38,7 +39,7 @@ resource managedIdentity 'Microsoft.ManagedIdentity/userAssignedIdentities@2018-
   location: location
 }
 
-resource roleAssignment 'Microsoft.Authorization/roleAssignments@2020-04-01-preview' = {
+resource roleAssignmentContributor 'Microsoft.Authorization/roleAssignments@2020-04-01-preview' = {
   scope: storageAccount
   name: guid(resourceGroup().id, managedIdentityName, storageAccountContributorRoleDefinitionId)

```

Comment:
still getting used to bicep - but I think this is going to be common in your samples?

For the roleAssignment you want to use the resourceId of the MSI in the name, not just the name of the resource.

Can you take a pass through the other samples and check those as well?

Reply:
Fixed. I will check the other samples and do PRs if needed. This is a common pattern I'm following but only where deployment scripts are necessary. If we can fix up the RPs so the scripts aren't necessary then I can probably remove them :)